vars/vars.yml.template

```diff
@@ -2,14 +2,15 @@
 bootstrap:
   ignHttpLocation: http://192.168.0.x:8080/ignition/bootstrap.ign
 set_proxy: false
-disconnected: false
+disconnected: {{ disconnected_setting }}
```

Comment:
Let's add all the possible variables to README

Reply:
Cool, I will update